addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1278,23 +1278,28 @@ registry.FullScreenHeight = publicWidget.Widget.extend({
      * @private
      */
     _adaptSize() {
-        const height = this._computeIdealHeight();
-        this.el.style.setProperty('min-height', `${height}px`, 'important');
+        const height = this._computeMainTopPosition();
+        this.el.style.setProperty('min-height', `calc(100dvh - ${height}px)`, 'important');
```

Comment:
I would also change the CSS class to use 100dvh (maybe with the fallback of 100vh?)